--- HandBrake-0.9.5/make/configure.py-orig	2011-03-14 05:48:41.338109218 +0100
+++ HandBrake-0.9.5/make/configure.py	2011-03-14 05:51:25.168107644 +0100
@@ -1228,7 +1228,7 @@
     class Tools:
         ar    = ToolProbe( 'AR.exe',    'ar' )
         cp    = ToolProbe( 'CP.exe',    'cp' )
-        curl  = ToolProbe( 'CURL.exe',  'curl', abort=False )
+        ##curl  = ToolProbe( 'CURL.exe',  'curl', abort=False )
         gcc   = ToolProbe( 'GCC.gcc',   'gcc', IfHost( 'gcc-4', '*-*-cygwin*' ))
 
         if host.match( '*-*-darwin*' ):
@@ -1243,13 +1243,13 @@
         ranlib = ToolProbe( 'RANLIB.exe', 'ranlib' )
         strip  = ToolProbe( 'STRIP.exe',  'strip' )
         tar    = ToolProbe( 'TAR.exe',    'gtar', 'tar' )
-        wget   = ToolProbe( 'WGET.exe',   'wget', abort=False )
+        ##wget   = ToolProbe( 'WGET.exe',   'wget', abort=False )
         yasm   = ToolProbe( 'YASM.exe',   'yasm', abort=False )
 
         xcodebuild = ToolProbe( 'XCODEBUILD.exe', 'xcodebuild', abort=False )
         lipo       = ToolProbe( 'LIPO.exe',       'lipo', abort=False )
 
-        fetch = SelectTool( 'FETCH.select', 'fetch', ['wget',wget], ['curl',curl] )
+        ##fetch = SelectTool( 'FETCH.select', 'fetch', ['wget',wget], ['curl',curl] )
 
     ## run tool probes
     for tool in ToolProbe.tools:
